validate/validate.go

```diff
@@ -256,28 +256,30 @@ func (v *Validator) CheckProcess() (msgs []string) {
 		}
 	}
 
-	for _, capability := range process.Capabilities {
-		if err := CapValid(capability, v.HostSpecific); err != nil {
-			msgs = append(msgs, fmt.Sprintf("capability %q is not valid, man capabilities(7)", capability))
+	if v.spec.Platform.OS == "linux" {
+		for _, capability := range process.Capabilities {
```

Comment:
But the validation of `capabilities` is compared with `CAPABILITIES (7)`,  [CAPABILITIES(7)](http://man7.org/linux/man-pages/man7/capabilities.7.html) is the value under linux.